.github/workflows/publish.yaml

```diff
@@ -0,0 +1,39 @@
+name: CI
+
+on: [pull_request, push]
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Snapcraft
+        uses: actions/checkout@v2
+        with:
+          # Fetch all of history so Snapcraft can determine its own version from git.
+          fetch-depth: 0
+
+      - name: Build Snapcraft Snap
+        id: build-snapcraft
+        uses: snapcore/action-build@v1
+
+      - name: Verify Snapcraft Snap
+        run: |
+          # Make sure it is installable.
+          sudo snap install --dangerous --classic ${{ steps.build-snapcraft.outputs.snap }}
+
+      - name: Publish Snapcraft Snap (PR)
+        if: ${{ github.event_name == 'pull_request' }}
+        uses: snapcore/action-publish@v1
+        with:
+          store_login: ${{ secrets.STORE_LOGIN }}
```

Comment:
IIRC, this secret won't be available to pull requests run from forks.  Maybe make the step conditional on `secrets.STORE_LOGIN != ''` too?

Reply:
Will do, thanks!